luigi/task.py

```diff
@@ -550,6 +551,37 @@ def on_success(self):
         Default behavior is to send an None value"""
         pass
 
+    @contextmanager
+    def _without_unpicklable_properties(self):
```

Comment:
given that the syntax for python's context manager invocation is `with xxx:`, perhaps change `without` to `no`. Contrast

```
with self._without_unpicklable_properties():
    ...
```

and


```
with self._no_unpicklable_properties():
    ...
```